orchagent/routeorch.cpp

```diff
@@ -696,6 +698,23 @@ void RouteOrch::doTask(Consumer& consumer)
                         ipv.resize(alsv.size());
                     }
 
+                    for (auto &vni_str: vni_labelv)
+                    {
+                        vni = static_cast<uint32_t>(std::stoul(vni_str));
+                        if (!m_vrfOrch->isL3VniVlan(vni))
```

Comment:
If the vrf-vni configuration is removed from OA and not from FRR after routes are installed then the routes still remain in the HW. 
Is this scenario handled ?

Reply:
In our case the VNI removal leads to error logs in SAI since it has references. This scenario again depends on references to maintain which I believe we don't do it. IMO I am fixing the scenarios which are easier to fix with the current implementation and then document the ones which are harder to fix.

Reply:
Currently SAI error is not creating orchagent crash since we are not handling status for this call. So we will get syslog error. For now I consider this a lower priority and document this.